.github/workflows/ci.yaml

```diff
@@ -1,6 +1,6 @@
 # THIS FILE WAS AUTOMATICALLY GENERATED, PLEASE DO NOT EDIT.
 #
-# Generated on 2024-02-02T15:47:02Z by kres cdedbc0-dirty.
+# Generated on 2024-02-04T14:11:21Z by kres undefined.
```

Comment:
first time seeing `undefined` as opposed to dirty :joy:

Reply:
That's when I run it in my IDE :). Anyways, now I did the regular way and it became dirty again :D